packages/create-plugin/src/commands/generate-actions/print-success-message.ts

```diff
@@ -12,7 +12,7 @@ export function printGenerateSuccessMessage(answers: CliArgs) {
     `- \`${packageManagerName} run dev\` to build (and watch) the plugin frontend code.`,
     ...(answers.hasBackend
       ? [
-          '- `mage -v build:linux` to build the plugin backend code. Rerun this command every time you edit your backend files.',
+          '- `mage` to build the plugin backend code. Rerun this command every time you edit your backend files.',
```

Comment:
```suggestion
          '- `mage -v build:backend` to build the plugin backend code. Rerun this command every time you edit your backend files.',
```

Reply:
You can run `mage -l` to list all available targets and their descriptions.

Reply:
that just generates the darwin binaries and still fails OOTB with `npm run server`

Reply:
hmm, sounds like a bug in the mage script. Can you give me some more details on what is happening?

Reply:
The issue with `build:backend` seems to building `gpx_m2mac_darwin_arm64` and `gpx_m2mac_darwin_amd64` but the docker container is looking for `gpx_m2mac_linux_arm64`.

An alternative would be to force the platform in the DockerFile to match the `build:linux` command.

```
FROM --platform=linux/amd64 grafana/${grafana_image}:${grafana_version}
```

Reply:
> The issue with `build:backend` seems to building `gpx_m2mac_darwin_arm64` and `gpx_m2mac_darwin_amd64` but the docker container is looking for `gpx_m2mac_linux_arm64`.
> 
> An alternative would be to pass the platform in the DockerFile to match the `build:linux` command.
> 
> ```
> FROM --platform=linux/amd64 grafana/${grafana_image}:${grafana_version}
> ```

Initially I thought it would cause trouble for users not having Rosetta installed to use the `--platform` prefix but after doing some research it seems like Rosetta is included in Big Sur and upwards. So all good in the hood 👍🏻

Reply:
Cool beans. Closing PR in favour of other solution